types/Student.ts

```diff
@@ -0,0 +1,12 @@
+export interface ProfileSetting {
+  description: string;
+  education: string;
+  goals: string;
+  experience: string;
+  school: string;
+  eductionLevel: string;
+}
+export default interface Student {
```

Comment:
The lack of a newline between these two really bothers me
```suggestion
}

export default interface Student {
```

Reply:
\n